docs/topics/db/aggregation.txt

```diff
@@ -323,6 +323,17 @@ title that starts with "Django" using the query::
 
     >>> Book.objects.filter(name__startswith="Django").aggregate(Avg('price'))
 
+When filtering against :ref:`multiple values <spanning-multi-valued-relationships>`,
```

Comment:
Does the reference here count against the character limit?  If so, I can wrap it.